addons/web/controllers/main.py

```diff
@@ -1585,6 +1586,49 @@ def get_app_url(self, req, app):
         sakey = Session().save_session_action(action)
         return werkzeug.utils.redirect('/web#sa={0}'.format(sakey))
 
+class AttachmentController(http.Controller):
```

Comment:
why not on the Class Binary of the same file?

@csnauwaert what do you think is best?